### DISCUSSION TITLE
Error writing out MNI warps: "SVD did not converge"

### DISCUSSION THREAD

**Discussion by jamesrcoleman97** [simnibs/simnibs#182] [7.Creating head models] [answered]
I've been running into issues using the charm command with the past couple of subjects I've tried running (only input is a single T1w scan). I keep getting the same error: "numpy.linalg.LinAlgError: SVD did not converge" (see log file below).

[charm_log.txt](https://github.com/simnibs/simnibs/files/12314507/charm_log.txt)

Any ideas on what might be going wrong? Thanks in advance for help on this!

**Comment by oulap** [comment, marked as answer]
I've seen something similar once before and the problem had something to do with the underlying math library versions. Can you try to see which libraries you're using by first opening a terminal, then firing up the simnibs python interpreter (simnibs_python) and then running:

`import numpy as np`
`np.show_config()`

Last time this problem was solved by installing numpy compiled against intel's libraries. You can try that by calling (in a terminal):

`simnibs_python -m pip install intel-numpy`

**Reply by jamesrcoleman97** [reply]
Thank you, that must've been the exact issue because charm ran perfectly just now! I ran np.show_config() and it looked like I was missing a few extensions:

> Supported SIMD extensions in this NumPy install:
    baseline = SSE,SSE2,SSE3
    found = SSSE3,SSE41,POPCNT,SSE42,AVX,F16C,FMA3,AVX2,AVX512F,AVX512CD,AVX512_SKX
    not found = AVX512_KNL,AVX512_KNM,AVX512_CNL
